src/walletSdk/Anchor/index.ts

```diff
@@ -179,15 +187,20 @@ export class Anchor {
 
     try {
       // TODO - use httpClient
-      const resp = await axios.get(`${transferServerEndpoint}/transactions?${queryString.stringify(apiParams)}`, {
-        headers: {
-          Authorization: `Bearer ${authToken}`,
-        },
-      });
+      const resp = await axios.get(
```

Comment:
And here

Reply:
https://github.com/stellar/typescript-wallet-sdk/pull/8/commits/d657eea6d56447f306ef5d2dd90e0bd3b815c7cc